build.gradle

```diff
@@ -134,6 +134,6 @@ spotless {
 		trimTrailingWhitespace();
 	}
 }
-tasks.named("build") { dependsOn("spotlessApply") }
-tasks.named("check") { dependsOn("spotlessApply") }
-tasks.named("deploy") { dependsOn("spotlessApply") }
+tasks.named("build") { finalizedBy("spotlessApply") }
+tasks.named("deploy") { finalizedBy("spotlessApply") }
```

Comment:
What was the rationale for switching to running spotlessApply after build and deploy? I don't have anything in particular against it, just wondering why.

Reply:
Since spotlesscheck doesn't execute in build/deploy, I just moved it after so that in case spotless has an error or something it'll happen after compilejava

Reply:
I see, for some reason I forgot about spotless crashes.